dl_binaries.sh

```diff
@@ -22,3 +22,6 @@ do
     curl -sS https://update.tabnine.com/$path > binaries/$path
     chmod +x binaries/$path
 done
+
```

Comment:
copy the `dl_binaries.sh` from the repo, as I've also added `rm -rf` to the old folder

Reply:
done